jetty-util/src/main/java/org/eclipse/jetty/util/IteratingCallback.java

```diff
@@ -403,7 +403,7 @@ public void failed(Throwable x)
 
     public void close()
     {
-        boolean failure=false;
+        String failure=null;
```

Comment:
The changes in `IteratingCallback` seems like a candidate for jetty-9.4.x backport.